### PR TITLE
Update new bug report template; delete old one

### PR DESCRIPTION
**Pull request by devalevenkatesh**
**Issue #:**

**Description of changes:**
- Update the added new bug report template to make browser as `input` and not as a `dropdown` since we would need to maintain this in sync with official browser supported list. Also, easier for builder to just specify it.
- Still, kept browser version separate since, builder may not provide version when clubbed together with browser.
- Removed older bug template.

**Testing**

1. Have you successfully run `npm run build:release` locally? NA.
2. How did you test these changes? NA.
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? NA.
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? NA.
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? NA.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by michhyun1 — APPROVED** [review]

**Review by michhyun1 — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 1]
```diff
@@ -1,50 +1,42 @@
-name: Bug Report
+name: Bug Report Form
 description: Report unexpected behavior during a meeting session here
 body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to fill out this bug report!
+        Please go through the existing [issues](https://github.com/aws/amazon-chime-sdk-js/issues) or [FAQs](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html) to see if your issue has already been addressed before.
```
Comment (ltrung) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:7]: Thought @vidya-mohan wants to convert this to checkbox?

File: .github/ISSUE_TEMPLATE/bug-report.yaml [hunk 2]
```diff
@@ -56,8 +48,15 @@ body:
...
     id: logs
     attributes:
-      label: Browser console logs.
-      description: Check our [debugging](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html#debugging) FAQ section to provide this required information. Please attach INFO-level logs with timestamps to the report from *before* the session started up until the point at which the unexpected behavior occurred. For multi-participant calls, attach a log file for each participant if possible.
+      label: Browser console logs
+      description: |
+        Check our [debugging](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html#debugging) FAQ section to provide this required information.
+        Please attach INFO-level logs with timestamps to the report from *before* the session started up until the point at which the unexpected behavior occurred.
+        For multi-participant calls, attach a log file for each participant if possible.
       render: shell
```
Comment (ltrung) [inline .github/ISSUE_TEMPLATE/bug-report.yaml:56]: Do we want them to paste the console log here or do attachment?

File: .github/ISSUE_TEMPLATE/bug-report.yaml (hunk 1 above)
Comment (devalevenkatesh) [reply .github/ISSUE_TEMPLATE/bug-report.yaml]: Thanks for the catch! I have converted them to checkboxes now.

File: .github/ISSUE_TEMPLATE/bug-report.yaml (hunk 2 above)
Comment (devalevenkatesh) [reply .github/ISSUE_TEMPLATE/bug-report.yaml]: Good question. Got to know if we specify the `render` option then users cannot attach the files. Removed that option.

Reference link: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#textarea

**Review by ltrung — APPROVED** [review]

**Pull request merged**